docs/TESTS.md

```diff
@@ -31,3 +31,83 @@ Or, you can enable all the tests by commenting out the
 ```elixir
 # ExUnit.configure trace: true
 ```
+
+### Typespecs and Dialyzer (DIscrepancy AnalYZer for ERlang programs)
+
+Elixir exercises include a skeleton implementation file. This
+file outlines the module and functions that you are expected
+to implement. In most exercises, you will find typespecs
+above the function declaration. These start with the `@spec` tag and
+typically follow the `@spec function_name(type1, type2) :: return_type`
+format. These are used in Elixir and Erlang as documentation and
+in conjunction with a tool called Dialyzer to find type inconsistencies
+and possible bugs. For more information see the
+[Elixir typespecs guide](http://elixir-lang.org/getting-started/typespecs-and-behaviours.html)
+or the [typespecs documentation](http://elixir-lang.org/docs/stable/elixir/typespecs.html).
+
+If you are a more experienced Elixir user, you may want to check
```

Comment:
OK. What I feel we need to stress is that unlike the tests, running Dialyzer and getting rid of any warnings is optional. New users should not feel they have to take this step before submitting their solutions.